### ISSUE TITLE
Start TE View does not respect stopped time entries

### ISSUE THREAD

**Issue by kornelijepetak** [toggl/mobileapp#2061]
Steps to reproduce:

* Start a new TE
* Click on a clock icon (to change the `Start and stop times`)
* Click on the `Stop` button as to stop the time entry.
* _The stop time shows correctly on the dialog_
* Click `Save`
* _User is returned to the start time entry_
* 🐛 _The duration is still going on at the top of the screen.
* 🗒 Note: The time entry **is** stopped, however. If you save the TE, it will be in the stopped mode, not still running.

Basically, only the UI should be changed to show that the time entry was stopped. Instead of duration, maybe display something like `06/14 16:22 - 16:28`. @pe0ny please advise 😄

**Comment by kornelijepetak** [comment]
This actually happens on both iOS and Android.
So, I'm adding Foundation tag as well cause it's probably not supported by the VM.
Then, when we know what we want to show in this case, the appropriate UI should be done for both platforms.

**Comment by pe0ny** [comment]
@kornelijepetak 
> Note: The time entry is stopped, however. If you save the TE, it will be in the stopped mode, not still running.

What do you mean by "f you save the TE, it will be in the stopped mode, not still running"? I'm not quite sure I get that.

**Comment by pe0ny** [comment]
Also, in the Start view I would just display the duration for now when time entry is stopped. I believe when it's not ticking, it's a good indicator that the entry isn't running anymore.

**Comment by kornelijepetak** [comment]
> What do you mean by "If you save the TE, it will be in the stopped mode, not still running"? I'm not quite sure I get that.

Yeah, maybe I explained this a bit vaguely.
The thing is, even if you clicked stop, the duration on the Start TE screen would still be ticking, so it would look like the entry **has not been stopped**. However, on `save` you could see that the entry has indeed be stopped. It's just that the UI shows ticking instead of the final duration.

**Comment by kornelijepetak** [comment]
> Also, in the Start view I would just display the duration for now when time entry is stopped. I believe when it's not ticking, it's a good indicator that the entry isn't running anymore.

Makes sense. Basically, this duration label still ticking is the core problem of this issue.
